packages/edit-site/src/components/dataviews/pagination.js

```diff
@@ -83,16 +83,17 @@ function Pagination( {
 										min={ 1 }
 										max={ totalPages }
 										onChange={ ( value ) => {
+											const _value = +value;
```

Comment:
This is the fix? Can you explain it. It's a bit obscure?

Reply:
The value of the input on the `onChange` callback is a `string` and we keep an integer in our view config. That made some strict checks to fail.

Reply:
Ah ok, maybe we should do `parseInt` instead of the `+` trick. (as you wish)